### ISSUE TITLE
Hugging Face Hub Integration

### ISSUE THREAD

**Issue by osanseviero** [POSTECH-CVLab/PyTorch-StudioGAN#111]
Hi!

This project is very nice. I saw that at the moment you're sharing the model weights via Google Drive links, but I think it would be very nice if these were shared via the [Hugging Face Hub](http://hf.co/models) as well.

Some of the benefits of sharing your models through the Hub would be:
* repos provide useful metadata about their tasks, languages, metrics, etc that make them discoverable
* [Demos](https://huggingface.co/spaces) and widgets that are freely hosted and allow anyone to try out the models
* versioning, commit history and diffs
* multiple features from TensorBoard visualizations, PapersWithCode integration, and more
* wider reach of your work to the ecosystem

Creating the repos and adding new models should be a relatively straightforward process if you've used Git before. This is a [step-by-step guide](https://huggingface.co/docs/hub/adding-a-model) explaining the process in case you're interested. Please let us know if you would be interested and if you have any questions. We would also be happy to further aid you in building some demos out of this.

Happy to hear your thoughts,
**Omar and the Hugging Face team**

cc @merveenoyan

**Comment by MINGUKKANG** [comment]
Hi, osanseviero

I think It would be nice to share the weights of various GANs using Hugging Face Hub.

I want to upload the weights right now, but I don't have enough time to do it. 

After completing urgent work, I will comment on this issue. 

Thank you 🤗

Best,

Minguk

**Comment by osanseviero** [comment]
Hi Minguk! I hope you're doing great!

I was wondering if you're still interested in sharing the model weights through the Hub (cc @NimaBoscarino)

**Comment by MINGUKKANG** [comment]
Yes, we are providing all model weights through [Hugging Face Hub](https://huggingface.co/Mingguksky/PyTorch-StudioGAN/tree/main). 

Thank you.
